src/pinmux.c

```diff
@@ -33,62 +36,133 @@
 //
 //*****************************************************************************
 
-// This file was automatically generated on 7/21/2014 at 3:06:20 PM
-// by TI PinMux version 3.0.334
+// This file was automatically generated on 2/13/2017 at 10:28:44 AM
+// by TI PinMux version 4.0.1482 
 //
 //*****************************************************************************
 
-#include "pinmux.h"
+#include "pinmux.h" 
 #include "hw_types.h"
 #include "hw_memmap.h"
 #include "hw_gpio.h"
 #include "pin.h"
```

Comment:
It's not needed. The "pinmux.h" header includes the declarations that we define in this source file. I included the "rom_pinmux.h" in the repository just so that we had that artifact from the pinmux tool.